LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 megabot
```

Comment:
Only small thing (which I may do in post if you don't want to) change megabot to CS Careers Hub

Reply:
CS Career Hub*

Reply:
```suggestion
Copyright (c) 2020 CS Career Hub
```